code/modules/SCP/SCPs/SCP-2427-3.dm

```diff
@@ -178,25 +178,25 @@ GLOBAL_LIST_EMPTY(scp2427_3s)
 		return
 	if(ishuman(A) && (satiety > min_satiety) && !(A in impurity_list))
 		var/mob/living/carbon/human/H = A
-		if(H.stat != DEAD)
+		if(H.stat != DEAD && !H.isMonkey())
 			to_chat(src, SPAN_WARNING("We cannot decide if they are pure or not just yet..."))
 			return
 	if(isliving(A))
 		var/mob/living/L = A
 		// Brute loss part is mainly for humans
-		if((L.stat == DEAD) || (L.stat && ((L.health <= L.maxHealth * 0.25) || (L.getBruteLoss() >= L.maxHealth * 4))))
+		if((L.stat == DEAD) || L.isMonkey() || (L.stat && ((L.health <= L.maxHealth * 0.25) || (L.getBruteLoss() >= L.maxHealth * 4)))) //This is an informational comment; but please do not remove the 'ISMONKEY' check due to the fact that unless we recode the purity mechanism, 2427-3 Cannot attack them!
 			var/nutr = L.mob_size
 			if(istype(L, /mob/living/simple_animal/hostile/retaliate/goat)) // Likes goats
 				nutr = round(max_satiety * 0.5)
 			if(ishuman(L))
 				nutr = round(max_satiety * 0.2)
 			if(L.isMonkey())
-				nutr = round(max_satiety * 0.05)
+				nutr = round(max_satiety * 0.1)
```

Comment:
No. This was done because monkey cubes cost extremely low; You are not meant to feed monkeys to it.